src/CompanyService.Data/PositionRepository.cs

```diff
@@ -43,6 +43,20 @@ public List<DbPosition> FindPositions()
             return _provider.Positions.ToList();
         }
 
+        public DbPosition GetUserPosition(Guid userId)
+        {
+            var dbCompanyUser = _provider.PositionUsers
+                .Include(u => u.Position)
+                .FirstOrDefault(pu => pu.UserId == userId);
+
+            if (dbCompanyUser == null)
+            {
+                throw new NotFoundException($"User with id: '{userId}' was not found.");
+            }
+
+            return _provider.Positions.Find(dbCompanyUser.PositionId);
```

Comment:
this look strange, may be simple write `dbCompanyUser .Position`? and why this method not place in PositionUserRepository?

Reply:
it was method which I forgot to delete